lib/temporal/workflow/execution_info.rb

```diff
@@ -26,8 +26,8 @@ def self.generate_from(response)
           workflow: response.type.name,
           workflow_id: response.execution.workflowId,
           run_id: response.execution.run_id,
-          start_time: Utils.time_from_nanos(response.startTime),
-          close_time: Utils.time_from_nanos(response.closeTime),
+          start_time: response.start_time.to_time,
```

Comment:
As far as I understand `response.start_time` is an instance of `Google::Protobuf::Timestamp`, which does not implement `#to_time`

Reply:
It's defined here I believe: https://github.com/protocolbuffers/protobuf/blob/master/ruby/lib/google/protobuf/well_known_types.rb#L74

Reply:
```ruby
>> timestamp = Google::Protobuf::Timestamp.new
>> timestamp.to_time

NoMethodError (undefined method `to_time' for <Google::Protobuf::Timestamp: seconds: 0, nanos: 0>:Google::Protobuf::Timestamp)
```

Reply:
Hmm. Will dig into that to understand why that's not present.

Reply:
Had to explicitly `require 'google/protobuf/well_known_types'` to make it work I don't think it is loaded by default

Reply:
Yep, found an issue discussing that. Updated my PR to require it.